hyperspy/conftest.py

```diff
@@ -36,10 +36,10 @@
 hs.preferences.Plot.cmap_navigator = 'viridis'
 hs.preferences.Plot.cmap_signal = 'viridis'
 
-def pytest_configure(config):
-    config.addinivalue_line(
-        "markers", "parallel: a test that is itself parallel and should be run serially."
-    )
+# Set parallel to False by default, so only
+# those tests with parallel=True are run in parallel
+hs.preferences.General.parallel = False
```

Comment:
I am not clear at all with what the `parallel` marker does and I am wondering if this is still necessary?

Reply:
This isn't a parallel marker - this sets the default behaviour of `.map(parallel=None)` and any other functions that use `.map()`. Setting it to `False` explicitly here means that any tests that might use parallel threads won't unless the test explicitly asks for it (`parallel=True`) to minimize conflicts with the parallelized test suite.

The parallel marker I think you're referring to has been removed entirely.

Reply:
Yes, the marker have been removed and this is why I am wondering whether it was really necessary or not?